projects/components/src/header/page/page-header.component.ts

```diff
@@ -23,7 +26,40 @@ import { NavigableTab } from '../../tabs/navigable/navigable-tab';
       class="page-header"
       [class.bottom-border]="!this.tabs?.length"
     >
-      <div [ngClass]="this.contentAlignment">
+      <!-- If PageTimeRange feature flag is enabled, consumer of this component must specify the type of content being
```

Comment:
https://github.com/Traceableai/traceable-ui/pull/3558/files#r823183765

Reply:
should we just do this via an input property?

Reply:
You either give it the same input whether the flag is assigned or not then resolve it into two impls inside the component, or each consumer needs to check the flag and assign different inputs depending on the flag (at that point, makes more sense to have different components).

Reply:
Turns out i was running into an actual angular bug that's been open since 2016  :expressionless: . https://github.com/angular/angular/issues/7795

Ditched the directives and fixed with inputs not being affected or needing change, and two code paths depending on the FF

Reply:
That doesn't sound like the same thing, but I'll qualify that with I'm still not quite sure what you were running into. My understanding is we were trying to mix content projection with templates, but regardless of whether they got projected, we never rendered those templates - that issue was with structural directives that handle rendering like ngIf - i.e. you can't use two ngIf siblings in the same content.